pybamm/expression_tree/unary_operators.py

```diff
@@ -634,7 +634,7 @@ def __init__(self, child, integration_variable):
         if isinstance(integration_variable, list):
             if len(integration_variable) > 1:
                 raise NotImplementedError(
-                    "Indefinite integral only implemeted w.r.t. one variable"
+                    "Indefinite integral only implemented w.r.t. one variable"
```

Comment:
For the [failing tests](https://github.com/pybamm-team/PyBaMM/actions/runs/3669113413/jobs/6202712976) -

The errors and warnings are tested using `unittest` in the `tests` directory. Changing the error message should be accompanied by making the same change in the tests. For this case, here -

https://github.com/pybamm-team/PyBaMM/blob/c38f385675c0a6481de776acd5de7182eca77ab8/tests/unit/test_expression_tree/test_unary_operators.py#L357-L360

Reply:
I had missed that. Thanks